code/game/machinery/computer/camera.dm

```diff
@@ -193,6 +197,11 @@
 			D["[C.c_tag]"] = C
 	return D
 
+/obj/machinery/computer/security/attackby(obj/item/bodycam/bc, mob/user, params)
+		bc.cameranetwork = network
+		user.balloon_alert(user, "body camera linked to network.")
+		return
+
```

Comment:
This runtime if you attack it with something that isent a body-cam and prevents you from attacking it
```suggestion
/obj/machinery/computer/security/attackby(obj/item/I, mob/user, params)
	if(istype(I, obj/item/bodycam))
		var/obj/item/bodycam/bodycam = I
		bodycam.cameranetwork = network
		user.balloon_alert(user, "body camera linked to network.")
	else
		return ..()
```